# Average Order Value (AOV) Analysis



**Objective:**
Calculate the Average Order Value (AOV) using the cleaned e-commerce dataset.

In [5]:
import pandas as pd

# Load cleaned dataset
df = pd.read_excel("cleaned_online_retail.xlsx")

# Create Revenue column
df["Revenue"] = df["Quantity"] * df["UnitPrice"]

# Calculate AOV metrics
total_revenue = df["Revenue"].sum()
total_orders = df["InvoiceNo"].nunique()
aov = total_revenue / total_orders

# Create summary table
results = pd.DataFrame({
    "Metric": ["Total Revenue", "Total Orders", "Average Order Value (AOV)"],
    "Value": [round(total_revenue, 2), total_orders, round(aov, 2)]
})

print("\nAOV ANALYSIS RESULTS")
print("-" * 40)
print(results.to_string(index=False))


AOV ANALYSIS RESULTS
----------------------------------------
                   Metric      Value
            Total Revenue 5384117.65
             Total Orders   15434.00
Average Order Value (AOV)     348.85


**Conclusion:**
The Average Order Value (AOV) was calculated using the cleaned e-commerce dataset. The analysis produced a total revenue of 5,384,117.65 from 15,434 unique orders, resulting in an AOV of 348.85. This metric represents the average amount spent per order and will be used as a key input for Customer Lifetime Value (CLTV) calculations in the next stage of the project.